WireUI/Sources/WireMoveToFolderUI/Views/FolderPicker/FolderPicker.swift

```diff
@@ -23,9 +23,17 @@ import WireReusableUIComponents
 public struct FolderPicker: View {
```

Comment:
I think we have both created `FolderPicker` which is a little confusing. Let's discuss naming tomorrow. I think one of us should rename our component.